Duplicati/Library/Main/Operation/FilelistProcessor.cs

```diff
@@ -281,7 +319,10 @@ orderby n.Time descending
                         if (remoteFound)
                         {
                             Logging.Log.WriteInformationMessage(LOGTAG, "RemoveUnwantedRemoteFile", "removing remote file listed as {0}: {1}", i.State, i.Name);
-                            await backendManager.DeleteAsync(i.Name, i.Size, true, CancellationToken.None).ConfigureAwait(false);
+                            if (options.Dryrun)
+                                Logging.Log.WriteDryrunMessage(LOGTAG, "WouldDeleteRemoteFile", "Would delete remote file: {0}", i.Name);
+                            else if (verifyMode != VerifyMode.VerifyOnly)
+                                await backendManager.DeleteAsync(i.Name, i.Size, true, CancellationToken.None).ConfigureAwait(false);

```

Comment:
Shouldn't `VerifyOnly` warn that the volume is marked for deletion? Or is that handled by another operation at a later point? I'm thinking that the verify only should report if anything is not as it should be.

Reply:
My logic for this is that the operations that send in `VerifyOnly` only needs to read the data. Even if there are partial uploads, it seems weird if a "list" or "restore" operation can delete remote files.

Since the files are "accepted partial files" they will be cleaned up on the next run that is not `VerifyOnly`.